programs/lb_clmm/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lb_clmm"
-version = "0.8.2"
+version = "0.8.5"
```

Comment:
why pump from .2 to .5? 
where is .3?

Reply:
sorry my mistake, need to revert all the bumps to 0.8.5. I will update the step bump in other PR